lib/kernel.ts

```diff
@@ -256,6 +266,62 @@ const patchContract = (
 	}, contract);
 };
 
+/**
+ * @summary Upsert a contract in an unsafe way (DANGEROUS)
+ * @function
+ * @public
+ *
+ * @description
+ * This bypasses the whole authorization system, so use with care.
+ *
+ * This function has the added limitation that you can only insert
+ * contracts of types that are defined in the Jellyfish core.
+ *
+ * @param {Object} context - exectuion context
+ * @param {Object} backend - backend
+ * @param {Object} contract - contract
+ * @returns {Object} contract
+ *
+ * @example
+ * const contract = await unsafeUpsertContract(backend, {
+ *   type: 'foo',
+ *   links: {},
+ *   requires: [],
+ *   capabilities: [],
+ *   tags: [],
+ *   active: true,
+ *   data: {
+ *     foo: 'bar'
+ *   }
+ * })
+ *
+ * console.log(contract.id)
+ */
+export const unsafeUpsertContract = async (
+	context: Context,
+	backend: DatabaseBackend,
+	contract: Contract,
+): Promise<Contract> => {
+	jsonSchema.validate(
+		VERSIONED_CONTRACTS[CONTRACT_CONTRACT_TYPE].data.schema as any,
+		contract,
+	);
+	jsonSchema.validate(
+		VERSIONED_CONTRACTS[contract.type].data.schema as any,
+		contract,
+	);
+	return backend.upsertElement(context, contract);
+};
+
+/** @deprecated */
+export const unsafeUpsertCard = async (
```

Comment:
Can remove this as it's not part of the public API

Reply:
I did a quick search and it seems that no other modules are using it. Could you confirm this?

Reply:
Yep, can be removed